.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report to help us fix or improve an issue
+---
+
+## Describe the bug
+
+_A clear and concise description of what the bug is, and the conditions when it occurs._
+
+## How to reproduce
+
+_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._
+
+Steps to reproduce the behavior:
+
+1.  Go to '...'
+2.  Click on '....'
+3.  Scroll down to '....'
+4.  See error
+
+## Expected behavior
+
+_A clear and concise description of what you expected to happen._
+
+## Versions
+
+**Clarity project:**
+
+- [ ] Clarity Core
+- [ ] Clarity Angular/UI
+
+**Clarity version:**
+
+- [ ] v5.x
+- [ ] v12.x
+- [ ] v13.x
+- [ ] Other:
+
+**Framework:**
```

Comment:
I also don't think we need the Framework and Framework versions because we should know that given the Clarity version above.

Reply:
I would recommend leaving this because clr-ui still exists in the repo, not all issues filed are matched to the clarity version. Also its useful to know the framework versions precisely not just the major

Reply:
Okay, I can get on board with that